docs: update deployment section (#259)
wasmCloud/wasmCloud.com: docs/deployment/hosts/local-artifacts.mdx

---
title: "Local Artifacts"
description: "A quick note on best practices for starting applications from local artifacts"
sidebar_position: 4
---

When running wasmCloud locally using `wash up`, it's simple to deploy applications using local file references. This enables rapid iteration and testing of applications during development, by not having to push and pull artifacts to a registry. In a production environment, however, deploying artifacts from a filesystem is not always wise, as local artifacts are not immutable and may be tampered with.

> Fortunately, **the wasmCloud host denies requests to start actors and providers from local artifacts** by default. `wash up` specifically overrides this behavior and enables loading from file references via the `--allow-file-load` [option](/docs/reference/host-config) on the host. If for some reason it is necessary to start from a file reference in production, this option can be enabled.

Reviewer: ```suggestion
Fortunately, **the wasmCloud host denies requests to start actors and providers from local artifacts** by default. To enable loading from file references with `wash up`, you can use the `--allow-file-load` [option](/docs/reference/host-config) on the host. If for some reason it is necessary to start from a file reference in production, this option can be enabled.
```
better? I'm not sure

Author: No, this suggestion reads as if you need to set `--allow-file-load` when using `wash up`, whereas I'm trying to say that wash up sets `--allow-file-load` for you. Will rephrase though, agree it's awkward 🙏
